Entities/Items/Crate/Crate.as

```diff
@@ -294,11 +294,15 @@ void GetButtonsFor(CBlob@ this, CBlob@ caller)
 	}
 	else if (isUnpacking(this))
 	{
-		caller.CreateGenericButton("$DISABLED$", buttonpos, this, this.getCommandID("stop unpack"), getTranslatedString("Stop {ITEM}").replace("{ITEM}", getTranslatedString(this.get_string("packed name"))));
+		CButton@ button = caller.CreateGenericButton("$DISABLED$", buttonpos, this, this.getCommandID("stop unpack"), getTranslatedString("Stop {ITEM}").replace("{ITEM}", getTranslatedString(this.get_string("packed name"))));
```

Comment:
This line is pretty long and hard read. I suggest you move the button text out into a variable since that itself is quite long. Please do the same for the other line you added further down.
```suggestion
		string text = getTranslatedString("Stop {ITEM}").replace("{ITEM}", getTranslatedString(this.get_string("packed name")));
		CButton@ button = caller.CreateGenericButton("$DISABLED$", buttonpos, this, this.getCommandID("stop unpack"), text);
```

Reply:
While applying your change, I noticed the item name is not getting translated.
`!crate catapult` will say "Depaqueter catapult" instead of "Depaqueter catapulte" in french.
Isn't getTranslatedString() supposed to translate the string to the local language?

Reply:
That's because you spawned it with ``!crate`` and didn't set the name. Translation doesn't work with wrong capitals